test/Interop/Cxx/symbolic-imports/print-symbolic-module-interface.swift

```diff
@@ -0,0 +1,66 @@
+// RUN: rm -rf %t
+// RUN: split-file %s %t
+// RUN: %target-swift-ide-test -print-module -module-to-print=CxxModule -I %t/Inputs -source-filename=x -enable-experimental-cxx-interop -enable-experimental-feature ImportSymbolicCXXDecls | %FileCheck %s
+
+// REQUIRES: asserts
```

Comment:
Why the asserts requirement?

Reply:
it's an experimental feature not on in release mode (when enabled using `-enable-experimental-feature`).

Reply:
Ah, I didn't realize experimental features required asserts. Thanks!